src/controllers/profile-settings.controller.js

```diff
@@ -0,0 +1,64 @@
+import ResponseService from '../services/response.service';
+import UserService from '../services/user.service';
+import JwtService from '../services/jwt.service';
+/**
+ *
+ *
+ * @class ProfileSettingsController
+ */
+class ProfileSettingsController {
+  /**
+  * users can create an account
+  * @static
+  * @description POST /api/view-profile
+  * @param {object} req request object
+  * @param {object} res response object
+  * @memberof ProfileSettingsController
+  * @returns {object} ResponseService
+  */
+  static async viewProfile(req, res) {
+    const userToken = JwtService.verifyToken(req.token);
+    const userRegistered = await UserService.findUserByProperty({ email: userToken.email });
+
+    delete userRegistered.dataValues.password;
```

Comment:
Please use destructuring for the values you need. or try to refactor this in your way, I think the lines are repetitive in below functions and it might affect maintainability